README-pt.md

```diff
@@ -31,20 +31,20 @@ Na sua máquina local:
-4. Depois de efetuar login na Conversation Tool, clique no botão !["Import"](/doc/source/images/import_icon.png?raw=true).
-5. Importe o arquivo do Conversation [`workspace.json`](data/workspace.json) localizado no seu clone deste repositório. 
+2. Crie uma instância de serviço do [Assistant](https://console.ng.bluemix.net/catalog/services/Assistant/).
+3. Quando vir os serviços no painel, selecione o serviço Assistant que foi criado e clique no botão !["Launch Tool"](/doc/source/images/workspace_launch.png?raw=true).
+4. Depois de efetuar login na Assistant Tool, clique no botão !["Import"](/doc/source/images/import_icon.png?raw=true).
+5. Importe o arquivo do Assistant [`workspace.json`](data/workspace.json) localizado no seu clone deste repositório. 
 ## 3. Desenvolvimento e execução 
 Se você seguiu as etapas anteriores, já deve estar dentro do seu clone local e pronto para começar a executar o aplicativo no Unity. 
 1. `git clone https://github.com/watson-developer-cloud/unity-sdk.git` 
 2. Abra o Unity e, dentro do ativador de projeto, selecione o botão ![Open](doc/source/images/unity_open.png?raw=true). 
 3. Navegue até o local onde clonou este repositório e abra o diretório “Creation Sandbox”. 
 4. Se for exibido um prompt pedindo para fazer upgrade do projeto para uma versão mais recente do Unity, faça isso. 
 5. Siga [estas instruções](https://github.com/watson-developer-cloud/unity-sdk#getting-the-watson-sdk-and-adding-it-to-unity) para incluir no projeto o SDK do Watson Unity transferido por download na 1ª etapa. 
-6. Siga [estas instruções](https://github.com/watson-developer-cloud/unity-sdk#configuring-your-service-credentials) para incluir suas credenciais dos serviços Speech To Text e Conversation (localizadas no [IBM Bluemix](https://console.ng.bluemix.net/)). 
+6. Siga [estas instruções](https://github.com/watson-developer-cloud/unity-sdk#configuring-your-service-credentials) para incluir suas credenciais dos serviços Speech To Text e Assistant (localizadas no [IBM Bluemix](https://console.ng.bluemix.net/)). 
 7. Selecione `Advanced Mode` na janela de configuração.
-8. Clique em `Add Variable` e nomeie sua nova variável `ConversationV1_ID` e, em seguida, defina seu valor como o ID da área de trabalho do Conversation. ![Variable Configuration Example](doc/source/images/add_variable.png?raw=true) Para localizar o ID da área de trabalho, selecione o menu de expansão na área de trabalho do Conversation e selecione `View details`. 
+8. Clique em `Add Variable` e nomeie sua nova variável `AssistantV1_ID` e, em seguida, defina seu valor como o ID da área de trabalho do Assistant. ![Variable Configuration Example](doc/source/images/add_variable.png?raw=true) Para localizar o ID da área de trabalho, selecione o menu de expansão na área de trabalho do Assistant e selecione `View details`. 
```

Comment:
i'm not sure about this one, @scottdangelo ?

Reply:
Yes, this is a comment about a variable and cannot be changed.